docs/scip/0004-data-loading.md

```diff
@@ -0,0 +1,92 @@
+# Template for new SCIPs
+
+## Metadata
+
+Editor:
+  ...
+
+Status (raw | draft | stable | deprecated | retired):
+  raw
+
+## Description
+
+...
+
+## Requirements
+
+### What is included in the catalog entry for a datasource?
+
+- A URL to a remote location (as given below)
+  - The URL should be a browsable location, structured according to one of the supported 'data-sharing patterns' (see below)
+  - 
+
+- An indication of additional scivision plugins required to load the data, if not (?)
+
+### Image formats
+
+- Built-in support for any common format (via a library, such as skimage)
+- Built-in support for formats common across scientific domains, not included in the above
+  - Whether to support a given format should be considered against the cost of the additional dependencies it requires, and the burden of these (e.g. something that makes core scivision less portable, or adds an extra installation step might be rejected, but a single python-only dependency considered acceptable)
+
+- A 'plugin' system for extending to additional formats
+
+#### Additional image formats
+
+Below is a list of additional image formats to consider for built-in support
+
+-
+-
+
+
+### Supported data services
+
+#### Notes
+
+- 'Core' scivision (without additional) should maintain support for several remote data is commonly archived.
+
+- Often locations are specified using URLs with a http/https scheme, but e.g. directory browsing is not supported by http, which limits the generality or usefulness of this approach.
+
+- One possibility that is supported by plain http is a direct link to 'archive' file system (e.g. a zip file containing one of the patterns below).
```

Comment:
This way does seem like it would be easy for a data provider to comply with

Reply:
This is at least partly supported already, but it seemed worth being explicit about the issue, and that a zip file (or other archive) is available as a single-file option.  One motivation for having special handling of other data hosting services is so data providers have a better chance of including their data as-is, but with this option as a fallback, potentially.